pkg/bgpv1/manager/reconcilerv2/neighbor.go

```diff
@@ -264,6 +303,49 @@ func (r *NeighborReconciler) Reconcile(ctx context.Context, p ReconcileParams) e
 	return nil
 }
 
+// getDefaultGateway returns the default gateway address with lower priority using route and device
+// statedb tables and the provided default gateway configuration.
+func (r *NeighborReconciler) getDefaultGateway(defaultGateway *v2.DefaultGateway) (string, error) {
+	var defaultRoute netip.Prefix
+	switch defaultGateway.AddressFamily {
+	case "ipv4":
+		defaultRoute = netip.PrefixFrom(netip.IPv4Unspecified(), 0)
+	case "ipv6":
+		defaultRoute = netip.PrefixFrom(netip.IPv6Unspecified(), 0)
```

Comment:
I see some repetition for this `PrefixFrom` in this PR. Let's make them a variable.

```
var (
    ipv4Default = netip.PrefixFrom(netip.IPv4Unspecified(), 0)
    ipv6Default = netip.PrefixFrom(netip.IPv6Unspecified(), 0)
)
```